Unosquare.CodeChallenge/WarehouseServices/Services/CompaniesServices.cs

```diff
@@ -8,50 +8,91 @@
 using WarehouseServices.Contractor;
 using WarehouseCoreLib.Base;
 using WarehouseRepositories.Repositories;
+using Microsoft.EntityFrameworkCore;
 
 namespace WarehouseServices.Services
 {
     public class CompaniesServices : Service<Company>, IWarehouseService<CompaniesServices>
     {
-        private CompanyRepository repository;
+        private WarehouseDbContext dbcontext;
```

Comment:
Dependency Injected fields should be readonly, unless the value needs to change, this is not the case for the DbContext

Reply:
got it.. ill make the chances.

Reply:
the change was made on the commit 85fa3a

Reply:
sorry commit 85fa3a2